e2e-test/e2e/chaos/stresschaos/memory.go

```diff
@@ -21,7 +21,7 @@ import (
 	"net/http"
 	"time"
 
-	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
Why don't we globally import this package?

Reply:
My linter suggested it to me.